src/patternfly/components/Page/page.scss

```diff
@@ -273,35 +284,18 @@
   margin-left: auto; // to push it to the right
 
   .pf-m-icons {
-    display: none;
-    visibility: hidden;
-
-    @media screen and (min-width: $pf-global--breakpoint--lg) {
-      display: block;
-      visibility: visible;
-    }
-  }
-
-  .pf-m-mobile {
-    margin-right: 0;
-
-    @media screen and (min-width: $pf-global--breakpoint--lg) {
-      display: none;
-      visibility: hidden;
+    .pf-c-button {
```

Comment:
@srambach @mcoker added the `@includes pf-hidden-visible`  here, what do you think?